.circleci/config.yml

```diff
@@ -1729,6 +1729,22 @@ jobs:
             export PYTHONPATH=.
             ./scripts/send-benchmark-completion-notification-to-slack.py /tmp/codespeed-screenshots/
 
+  smoke-standalone-35-rate-limit:
+    working_directory: ~/scalyr-agent-2
+    environment:
+      SCALYR_MAX_SEND_RATE_ENFORCEMENT: "500KB/s"
```

Comment:
Since we define those values on the Circle CI environment variable level, test won't work correctly if you run it locally directly using tox, right?

If so, it may be better to put those values inside the tox target (``setenv``) or inside the code itself - this way tests should be reproducible no matter how you run them (via Circle CLI, using tox or directly using pytest binary).